### ISSUE TITLE
SwiftUI onAppear doesn't update UI

### ISSUE THREAD

**Issue by poopsnaggle** [rechsteiner/Parchment#469]
Hello,
Im using SwiftUI but it seem PageView doesn't update UI after onAppear perform.
Thankyou!
```
struct ContentView: View {
    @State var pagingItems = [PagingIndexItem]()
    
    var body: some View {
        PageView(items: pagingItems) { item in
            Text(item.title)
                .font(.largeTitle)
                .foregroundColor(.gray)
        }.onAppear {
            self.pagingItems = [
                PagingIndexItem(index: 0, title: "View 0"),
                PagingIndexItem(index: 1, title: "View 1"),
                PagingIndexItem(index: 2, title: "View 2"),
                PagingIndexItem(index: 3, title: "View 3")
            ]
        }
    }
}
```

**Comment by rechsteiner** [comment]
Hi @poopsnaggle. Thanks for the report! This should be fixed in the `master` branch now. Will release a new version in not so long

**Comment by rechsteiner** [comment]
This is available in the `2.4.0` release now 🙂
